workflow-templates/im-test-checkmarx-cli-sast-scan.yml

```diff
@@ -38,22 +37,25 @@ jobs:
   scan-the-code:
     runs-on: [self-hosted, ubuntu-20.04]
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout main for Schedule Trigger
+        uses: actions/checkout@v2
         if: github.event_name == 'schedule'
 
-      - uses: actions/checkout@v2
+      - name: Checkout Provided Ref for Repository Dispatch Trigger
+        uses: actions/checkout@v2
         if: github.event_name == 'repository_dispatch'
         with:
           ref: ${{ github.event.client_payload.ref }}
 
-      - uses: actions/checkout@v2
+      - name: Checkout provided branch/tag/sha for Workflow Dispatch Trigger
+        uses: actions/checkout@v2
         if: github.event_name == 'workflow_dispatch'
         with:
           ref: ${{ github.event.inputs.branch-tag-sha }}
 
       - name: Run the Scan
         id: scan
-        uses: im-open/checkmarx-cli-sast-scan@v2.0.0
+        uses: im-open/checkmarx-cli-sast-scan@v2.0.1
```

Comment:
Every other change is pretty innocuous, but is this one worth incrementing the version of the workflow for?

Reply:
Sorry I didn't see your question here until just now.  Yes I should have incremented the version for that change.